internal/services/formatters/csharp/dotnet_cli/formatter_test.go

```diff
@@ -32,23 +33,26 @@ import (
 )
 
 func TestParseOutput(t *testing.T) {
+	t.Cleanup(func() {
+		_ = os.RemoveAll(t.TempDir())
```

Comment:
t.TempDir is already cleaned at the end of test, I think that this is unnecessary.